src/test/java/HomeWork20.java

```diff
@@ -0,0 +1,40 @@
+import com.fasterxml.jackson.databind.JsonSerializable;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class HomeWork20 extends BaseTest {
+    @Test
+    public void deletePlaylist() throws InterruptedException {
+
+        String deletedPlaylistMsg = "Deleted playlist \"karina playlist.\"";
+        provideEmail("karina.usmanova01@testpro.io");
+        providePassword("YrEdlRVe");
+        clickLoginButton();
+
+        clickOnPlaylist();
+        clickDeleteThisPlaylist();
+        Thread.sleep(2000);
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

Reply:
Good day,
The thing is without  Thread.sleep I am getting an error and my test is failed.
The report shows: java.lang.AssertionError: expected [Deleted playlist "karina playlist."] but found []
It seems like its not able to read a notification pop

Sometimes my code has passed, sometimes test failed. I don't understand why. Screenshots attached. The same code...one time passed, one time failed.
<img width="1710" alt="Screenshot 2024-04-02 at 4 19 38 PM" src="https://github.com/Test-Pro-Bootcamp/A57-QA-Automation-Framework/assets/163922153/d033e67d-0012-4fc2-a861-81467f4539ce">
<img width="1710" alt="Screenshot 2024-04-02 at 4 20 26 PM" src="https://github.com/Test-Pro-Bootcamp/A57-QA-Automation-Framework/assets/163922153/dadae46d-d8fd-4476-bcfe-058d77f6020c">